### ISSUE TITLE
Amazon ship should have more visible player colour

### ISSUE THREAD

**Issue by tothxa** [widelands/widelands#6096]
**Is your feature request related to a problem? Please describe.**
The player colour of the Amazon ship is not visible. Only some wrinkles of the crew's dresses are coloured.

**Describe the solution you'd like**
The player colour should be obvious when looking at a ship. The entire dress should be coloured.

**Describe alternatives you've considered**
Alternatively, the domes on the boats making up the ship could be coloured.

**Additional context**
Haven't checked in-game, but judging by the `_pc` image, ferries are probably the same, though less critical because they can only work on owned waters.
